src/app/plugins/platform/Docker/plugins/cms/Drupal8/installDrupal.ts

```diff
@@ -14,6 +15,11 @@ export type PantheonProject = typeof pantheonProject;
 
 export type Project = PantheonProject | DrupalProject;
 
+// Define the debugging namespace to align with other debugger output.
+const debugNamespace =
+  'web-starter:app:plugins:platform:Docker:plugins:cms:Drupal8:installDrupal';
```

Comment:
Is this the namespace convention that Yeoman uses for composed generators?

Reply:
Yes it is